identity-platform/gke/backend/Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY requirements.txt .
 RUN pip install -r requirements.txt
 WORKDIR /app
 COPY . .
-CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 app:app
+CMD ["exec", "gunicorn", "--bind :$PORT", "--workers 1", "--threads 8", "app:app"] 
```

Comment:
This may change whether the PORT variable is interpolated at build-time vs. run-time, needs to be verified.

Reply:
I believe this won't work, should I revert and turn off this notification?